tests/FSharp.Compiler.Service.Tests/FSharp.Compiler.Service.Tests.fsproj

```diff
@@ -2,8 +2,7 @@
 
   <PropertyGroup>
     <OutputType>Exe</OutputType>
-    <TargetFrameworks Condition="'$(OS)' != 'Unix'">net472;net5.0</TargetFrameworks>
-    <TargetFrameworks Condition="'$(OS)' == 'Unix'">net5.0</TargetFrameworks>
+    <TargetFrameworks>net472;net5.0</TargetFrameworks>
```

Comment:
I think, this would fail to build on Linux/macOS, since net472 is not supported there.

Reply:
this doesn't have to be the case, if the .net framework reference assemblies nuget package was added as a dependency then these projects could at least build on non-windows.